docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -1110,6 +1111,34 @@ This will override the default directives as specified in the ``compiler_directi
 Note that explicit per-file or local directives as explained above take precedence over the
 values passed to ``cythonize``.
 
+.. _cline_in_traceback:
+
+C line numbers in tracebacks
+============================
+
+To provide more detailed debug information, Python tracebacks of Cython modules
+show the C line where the exception originated (or was propagated). This feature is not
+entirely for free and can visibly increase the C compile time as well as adding 0-5% to the
+size of the binary extension module. It can therefore be disabled using a C macro.
```

Comment:
What I am missing is any mention what is default setting. We can use something like this:

```suggestion
size of the binary extension module. Therefore it is disabled by default. Cython provides C macro to control C lines numbers in tracebacks:
```

Reply:
I've added a mention of the default (but not this text, because the default is to use the runtime check)